### PR TITLE
Adds Execution Sword

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
Adds a shitty traitor item worth 1tc that lets you decapitate people after 30 seconds of standing still. It broadcasts your location, you can dedicate it to a faction, and it plays one of two nasheeds. It deals less damage than the e-pen unless force code was changed that's why it's so cheap.

### CODE REVIEW

File: code/modules/assembly/bomb.dm [hunk 1]
```diff
@@ -149,6 +149,7 @@
 	var/strength = 1
 
 	var/turf/ground_zero = get_turf(loc)
+	bomb_mixture.temperature += 10 //you might ask, why this? lol
```
Comment (TheChosenEvilOne) [inline code/modules/assembly/bomb.dm]: ?

File: code/_globalvars/misc.dm [hunk 2]
```diff
@@ -34,3 +34,6 @@ GLOBAL_VAR_INIT(glowshrooms, 0)
 
 /// If drones are blacklisted from certain sensitive machines
 GLOBAL_VAR_INIT(drone_machine_blacklist_enabled, TRUE)
+
+//the executioner sword
+GLOBAL_VAR_INIT(nasheed_playing, FALSE)
```
Comment (TheChosenEvilOne) [inline code/_globalvars/misc.dm]: just move this to the executioners file.

**Review by TheChosenEvilOne — CHANGES_REQUESTED** [review]

File: code/modules/assembly/bomb.dm (hunk 1 above)
Comment (karmaisblackandbluepilled) [reply code/modules/assembly/bomb.dm:152]: strange, I could've swore I undid this when I changed the branch, hold on

File: code/game/objects/items/melee/executioners.dm [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+			nasheed.repeat = 0
+			nasheed.status = SOUND_STREAM
+			nasheed.volume = 100
+			for(var/mob/M in GLOB.player_list)
+				if(M.client.prefs.toggles & SOUND_MIDI)
+					var/user_vol = M.client.admin_music_volume
+					if(user_vol)
+						nasheed.volume = 100 * (user_vol / 100)
+					SEND_SOUND(M, nasheed)
+					nasheed.volume = 100
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/melee/executioners.dm]: this line isn't needed.
```suggestion
```

File: code/game/objects/items/melee/executioners.dm [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+			var/nasheed_chosen = pick(nasheed_list)
+			earrape_time = world.time + 250 //25 seconds between each
+			var/sound/nasheed = new()
+			nasheed.file = nasheed_chosen
+			nasheed.channel = CHANNEL_NASHEED
+			nasheed.frequency = 1
+			nasheed.wait = 1
+			nasheed.repeat = 0
+			nasheed.status = SOUND_STREAM
+			nasheed.volume = 100
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/melee/executioners.dm]: setting vars for no reason to default value.
```suggestion
			nasheed.wait = TRUE
			nasheed.status = SOUND_STREAM
```

File: code/game/objects/items/melee/executioners.dm [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+			for(var/mob/M in GLOB.player_list)
+				if(M.client.prefs.toggles & SOUND_MIDI)
+					var/user_vol = M.client.admin_music_volume
+					if(user_vol)
+						nasheed.volume = 100 * (user_vol / 100)
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/melee/executioners.dm]: user volume is always set and it goes from 0 to 1, not 0 to 100
```suggestion
					nasheed.volume = 100 * M.client.admin_music_volume
```

**Review by TheChosenEvilOne — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items.dm [hunk 6]
```diff
@@ -2065,3 +2065,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
 
+/datum/uplink_item/badass/execution_sword
+	name = "Executioners Sword"
+	desc = "This modified energy sword has been specially designed to cleanly remove the head of a human\
+			 being in one well aimed swipe. It contains a little hacked transmitter that will broadcast the\
+			 details of your gruesome execution on the Centcom announcement channel so everyone will know the\
+			 name of the filthy pig you are about to slaughter. You may dedicate your executions to whomever you\
+			 please by using the device in hand but you may only do so once. Be warned that you must remain still\
+			 for a long time to execute a target so be sure to have them restrained and if you should be interrupted\
+			 then news of your failure will be broadcast to the station."
```
Comment (TheChosenEvilOne) [inline code/modules/uplink/uplink_items.dm]: Linter doesn't like the indentation
```suggestion
	desc = "This modified energy sword has been specially designed to cleanly remove the head of a human \
			being in one well aimed swipe. It contains a little hacked transmitter that will broadcast the \
			details of your gruesome execution on the Centcom announcement channel so everyone will know the \
			name of the filthy pig you are about to slaughter. You may dedicate your executions to whomever you \
			please by using the device in hand but you may only do so once. Be warned that you must remain still \
			for a long time to execute a target so be sure to have them restrained and if you should be interrupted \
			then news of your failure will be broadcast to the station."
```

File: code/game/objects/items/melee/executioners.dm [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+	w_class = WEIGHT_CLASS_NORMAL
+	hitsound = 'sound/weapons/rapierhit.ogg'
+	var/execution_faction = "The Syndicate"
+	var/faction_chosen = FALSE
+	var/executing = FALSE
+	var/can_execute = TRUE
+	var/static/earrape_time = 0
+	var/nasheed_list = list('sound/misc/nasheed.ogg', 'sound/misc/nasheed2.ogg')
+
+obj/item/melee/execution_sword/attack_self(mob/living/user)
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/melee/executioners.dm]: pain
```suggestion
/obj/item/melee/execution_sword/attack_self(mob/living/user)
```

**Review by TheChosenEvilOne — APPROVED** [review]
I don't see anything else wrong, but just in case requesting review from Edge.

**Comment by EdgeLordExe** [comment]
looks good

**Pull request merged**